(AzureCXP) Fixes MicrosoftDocs/azure-docs-cli#2820

BhargaviAnnadevara-MSFT opened the pull request:
Added entry for `yamlc` output format

opbld32 commented:
Docs Build status updates of commit _[e5963a2](https://github.com/BhargaviAnnadevara-MSFT/azure-docs-cli/commits/e5963a2ddd9fd89e028122813a013d54fe707292)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/format-output-azure-cli.md](https://github.com/BhargaviAnnadevara-MSFT/azure-docs-cli/blob/patch-14/docs-ref-conceptual/format-output-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/format-output-azure-cli?branch=pr-en-us-2823) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C11%5Cb11f9ddf-791c-c17b-454e-2902f9b37938%5CPullRequest%5C202111110830501968-2823%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-12T08%3A32%3A44Z&sr=b&sp=r&sig=YQP5jO9Ytjd3xkntre89qZT6AYSdfa6kt1HPQM1494Q%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/b11f9ddf-791c-c17b-454e-2902f9b37938?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld32 commented:
Docs Build status updates of commit _[975daa1](https://github.com/BhargaviAnnadevara-MSFT/azure-docs-cli/commits/975daa1277119d3b5a9c593d6e81d0a23f5167b4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/format-output-azure-cli.md](https://github.com/BhargaviAnnadevara-MSFT/azure-docs-cli/blob/patch-14/docs-ref-conceptual/format-output-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/format-output-azure-cli?branch=pr-en-us-2823) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C11%5Cb11f9ddf-791c-c17b-454e-2902f9b37938%5CPullRequest%5C202111110845194611-2823%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-12T08%3A46%3A50Z&sr=b&sp=r&sig=xlsVjhf2HCyOtdBnKdvJtBcLSj8gf0bsXM1DChQhPUw%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/b11f9ddf-791c-c17b-454e-2902f9b37938?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

BhargaviAnnadevara-MSFT commented:
@dbradish-microsoft This PR addresses feedback provided as part of: https://github.com/MicrosoftDocs/azure-docs-cli/issues/2820. Please review and approve if it looks good to you.
